### ISSUE TITLE
Task: Create a DAAN mapping for schema.org

### ISSUE THREAD

**Issue by wmelder** [beeldengeluid/beng-lod-server#106]
- [ ] The dataset is published in a standardised format (Schema.org) (#3)
Note: This seems to be a more important requirement than before. The (internal) NISV schema is ideal for mapping all of our data. But for disseminating data is preferable to map the data to the (external) widely in use schema.org
- [x] Try to add content [negotiation by profile](https://www.w3.org/TR/dx-prof-conneg/), instead of separate route

**Comment by mwigham** [comment]
@wmelder can we tighten up the scope of this issue? We are not going to publish the entire dataset as schema in the near future (if ever). Perhaps we can limit it to a mapping for the CC0 fields? That would be appropriate for ODL

**Comment by wmelder** [comment]
@mwigham I see two options. What do you prefer? 
1) Relate the pull request you made for the schema.org persons, locations, etc. to this issue and close this issue once the PR is merged. 
2) Or we close this issue now and create a new issue for the PR. The mapping was created already (although not finished completely), so this issue can be closed anyway. 

For additional work (other cc0 fields) we can create task issues that are smaller and more precise and also relate these to future PR's.

I think I prefer the second.

**Comment by mwigham** [comment]
I'm happy with that one too.

**Comment by wmelder** [comment]
The work will continue in; https://github.com/beeldengeluid/beng-lod-server/issues/117
